package-system/AWSNativeSDK/FindAWSNativeSDK.cmake.Mac

```diff
@@ -88,59 +87,77 @@ function(ly_declare_aws_library)
 endfunction()
 
 
-#### Common ####
+#### CRT ####
 if(LY_MONOLITHIC_GAME)
     ly_declare_aws_library(
         NAME 
-            Common
-        LIB_FILE 
-            aws-c-common
-    )       
-else()
-    ly_declare_aws_library(
-        NAME 
-            Common
+            AWSCrt
         LIB_FILE 
-            aws-c-common
+            aws-crt-cpp
         BUILD_DEPENDENCIES
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.0unstable${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.1.0.0${CMAKE_SHARED_LIBRARY_SUFFIX}
-        RUNTIME_DEPENDENCIES
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.0unstable${CMAKE_SHARED_LIBRARY_SUFFIX}
-            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-common.1.0.0${CMAKE_SHARED_LIBRARY_SUFFIX}
-    )       
-endif()
-
-#### Checksums ####
-ly_declare_aws_library(
-    NAME 
-        Checksums
-    LIB_FILE 
-        aws-checksums
-)
-
-#### EventStream ####
-if(LY_MONOLITHIC_GAME)
-    ly_declare_aws_library(
-        NAME 
-            EventStream
-        LIB_FILE 
-            aws-c-event-stream
-        BUILD_DEPENDENCIES 
-            3rdParty::AWSNativeSDK::Checksums
+            ${AWSNATIVE_SDK_LIB_PATH}/libaws-c-s3.a
```

Comment:
Nit: Is .a the static library? Is there any Cmake parameters (might be something like ${CMAKE_STTATIC_LIBRARY_SUFFIX}) we can use here instead of hardcoded the suffix?